pulsar-broker/src/test/java/org/apache/pulsar/client/api/TopicReaderTest.java

```diff
@@ -554,7 +563,7 @@ public void testReaderIsAbleToSeekWithTimeOnBeginningOfTopic() throws Exception
         producer.close();
     }
 
-    @Test
+    @Test(timeOut = 5000)
```

Comment:
5seconds may seem a long time, but on the Apache Jenkins could be very little time :) 

BTW: there's already a default timeout on tests set on the testng config (I think it's 120 secs), to cover for stuck tests.

Reply:
I think 5 secs should be long enough to wait for this test. Do you want to leave it with default value or this explicit 5 secs?

Overall time taken by TopicReaderTests improves by half (from 30 secs to 15 secs in my machine) not so much but is something :)

Reply:
> I think 5 secs should be long enough to wait for this test. Do you want to leave it with default value or this explicit 5 secs?

Yes, on most machines it will take far less. On Apache Jenkins we've seen that things can get much slower and start causing spurious timeouts. I'd leave it at the default since it will not slowdown the tests. Timeout is just to prevent a test getting stuck to stop the job.

> Overall time taken by TopicReaderTests improves by half (from 30 secs to 15 secs in my machine) not so much but is something :)

That's nice!

Reply:
Great! fixed :+1: